bayesian/utility/v2/sa_response_builder.py

```diff
@@ -53,22 +59,25 @@ def get_response(self):
         stack_task_result = self._stack_result.get('task_result')
         stack_audit = stack_task_result.get('_audit', {})
 
-        return {
+        report = {
```

Comment:
Can you define a high level response model which holds toplevel fields along with recommendation and report.?